File: curso-udemy/curso-algoritmo-completo/Módulo 7 - Python/Exercícios/somaVetor.py
N: int = int(input("Quantos números serão inseridos? N: "))
vet: float = [0 for x in range(N)]
mediaFinal: float
somaVetor: float

print(f"Valores: ", end="")
for i in range(0,N):
    vet[i] = float(input("Digite um número: "))
    print(f"{vet[i]} ", end="")

print()

somaVetor = 0
for i in range(0,N):
    somaVetor += vet[i]
print(f"Soma Final: {somaVetor}")

mediaFinal = somaVetor / N
print(f"Média Final: {mediaFinal}")

File: programming-logic/algorithm/knowing-c/Exercícios/SOMAVETOR.c
#include <stdio.h>

int main(){

    int N, i;
    double somaVetor, mediaVetor;

    printf("Quantos números serão digitados?\nN: ");
    scanf("%d", &N);

    double vet[N];

    somaVetor = 0;
    mediaVetor = 0;

    for(i = 0; i < N; i++){
        scanf("%lf", &vet[i]);
    }

    for(i = 0; i < N; i++){
        somaVetor = somaVetor + vet[i];
    }

    mediaVetor = (double) somaVetor / N;

    printf("Valores: ");
    for(i = 0; i < N; i++){
        printf("%.1lf ", vet[i]);
    }
    printf("\n");
    printf("Soma: %.1lf", somaVetor);
    printf("\n");
    printf("Media: %.1lf", mediaVetor);

    return 0;
}
